d3rlpy/algos/base.py

```diff
@@ -228,8 +228,9 @@ def fit_online(
             logdir: root directory name to save logs.
             verbose: flag to show logged information on stdout.
             show_progress: flag to show progress bar for iterations.
-            tensorboard: flag to save logged information in tensorboard
-                (additional to the csv data)
+            tensorboard_dir: directory to save logged information in
+                tensorboard (additional to the csv data).  if None, the
```

Comment:
I'd like you to do close it with backslashes since this is visually better in the sphinx doc.
```
None -> ``None``
```